app/build/generated/source/kapt/debug/com/example/projetointegrador/database/AppDataBase_Impl.java

```diff
@@ -211,4 +231,18 @@ public ConfiguracoesDAO ConfiguracoesDAO() {
       }
     }
   }
+
+  @Override
+  public PaisMappingDAO PaisMappingDAO() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.